ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ip-igmp-snooping.j2

```diff
@@ -1,26 +1,61 @@
-{% if ip_igmp_snooping is defined and ip_igmp_snooping is not none %}
+{% if ip_igmp_snooping is arista.avd.defined %}
 
 ## IP IGMP Snooping
 
 ### IP IGMP Snooping Summary
 
-{%     if ip_igmp_snooping.globally_enabled is defined and ip_igmp_snooping.globally_enabled == false %}
-IGMP snooping is globally disabled
+| Global Settings | Values |
```

Comment:
Just use `Settings`. We are already under the IGMP Snooping context

Reply:
I will change it to Setting